### PR TITLE
Add the serviceAccount to privileged scc

### PR DESCRIPTION
**Pull request by MohamedAshiqrh**
Signed-off-by: Mohamed Ashiq Liyazudeen <mliyazud@redhat.com>

---
This change is [Reviewable](https://reviewable.io/reviews/gluster/gluster-kubernetes/204)

### CODE REVIEW

**Comment by humblec** [comment]
@jarrpa PTAL.

**Comment by humblec** [comment]
This used to be a prereq.:
` * Add the 'heketi-service-account' Service Account to the 'privileged' SCC`

**Comment by jarrpa** [comment]
Looks good. Please also remove line 314 (the comment Humble mentioned). :)

File: deploy/gk-deploy [hunk 1]
```diff
@@ -420,6 +420,7 @@ if [[ ${LOAD} -eq 0 ]]; then
       eval_output "${CLI} create -f ${TEMPLATES}/glusterfs-template.yaml 2>&1"
     fi
     eval_output "${CLI} policy add-role-to-user edit system:serviceaccount:${NAMESPACE}:heketi-service-account 2>&1"
+    eval_output "${CLI} adm policy add-scc-to-user privileged -z heketi-service-account"
```
Comment (raghavendra-talur) [inline deploy/gk-deploy]: Please look at line 314, we have that as a requirement. Should we leave it as a requirement instead of doing it in script?

Comment (raghavendra-talur) [inline deploy/gk-deploy]: I am late to the party, Jose and Humble have rightly mentioned the updates to be done.

**Comment by MohamedAshiqrh** [comment]
Done @humblec @jarrpa @raghavendra-talur 👍

**Comment by jarrpa** [comment]
LGTM

**Pull request merged**